### DISCUSSION TITLE
RATGDO Intermittently showing unavailable in Home Kit

### DISCUSSION THREAD

**Discussion by marshalisms** [ratgdo/homekit-ratgdo#178] [General]
When this happens, I am able to ping the IP of the RATGDO, but cannot pull up the web GUI and it shows offline in HomeKit.  I just set mine to reboot every few hours, hoping this will prevent this issue.  OIs anyone else having this issue?

**Comment by jgstroud** [comment]
There have been a few mentions of the web page being unresponsive, but not HomeKit.  This may be a regression in 1.3.5.  You might try 1.2.1 until we can root cause the issue

**Comment by marshalisms** [comment]
It was doing it on whatever version I installed about a month or so ago as
well. Not sure which version it was.
